tjreports/site/models/reports.php

```diff
@@ -512,6 +525,12 @@ protected function populateState($ordering = '', $direction = 'ASC')
 			$reportParams = $this->getReportParams($reportId);
 
 			$colToshow = $reportParams->get("colToshow");
+			$piiColumns = array_flip($reportParams->get("piiColumns"));
```

Comment:
Add comment for why flip is needed?

Reply:
**Requirement** - Remove piiColumns from colToshow.

**Problem** - colToshow object like

stdClass Object
(
    [id] => id
    [title] => title
    [created_by] => created_by
    [type] => type
    [access] => access
    [cat_title] => cat_title
)

and piiColumns array like

Array
(
    [0] => id
    [1] => title
)

And we need to remove id and title from colToshow object.

**Solution** -

1. Convert piiColumns array value to key using array_flip. like - Array ( [id] => 0, [title] => 1)
2. Using array_diff_key function remove same keys from colToshow

Reply:
i meant add in the code